crates/fluvio-cli/src/client/remote/list.rs

```diff
@@ -55,22 +61,27 @@ mod output {
     use fluvio_extension_common::output::TableOutputHandler;
     use fluvio_extension_common::t_println;
 
-    type ListVec = Vec<(String, String, String)>;
+    #[derive(Serialize)]
+    pub struct RemoteStatusRow {
```

Comment:
is this need to be public?

Reply:
I moved it to outside of this module, so it's needed. I did the same for `home/status.rs` too.